modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -80,3 +80,20 @@ option, and that is *can edit*.
 image:configuration/files/federation-3.png[image]
 +
 You may disconnect the share at any time by clicking the btn:[trash can] icon.
+
+== Known Issues
+
+=== Persistent Locks Are Not Guaranteed
+
+There is a known bug propagated persistent locks to federated instances.
+If a user creates an exclusive lock on a share, no other users should be able to modify it, nor its contents, and all users should see a lock icon on the share.
+
+However, this isn't the case.
+The following functionality has been recorded:
+
+* The user who created the lock sees the lock icon throughout the share.
+* The top-level of the share for receivers shows the lock icon.
+* Sub-items of the share *do not show the lock icon*.
+* The share and the share and its contents *can still be modified by all users*; specifically: 
```

Comment:
`The share and the share` seems to be doubled

Reply:
Thanks for spotting that one.